deployment/.gitignore

```diff
@@ -0,0 +1,6 @@
+venv/
```

Comment:
should we have this file here?

Reply:
No, that only made sense in the separate repo.